### ISSUE TITLE
Plugin broken with latest version of devpi-server

### ISSUE THREAD

**Issue by apatrushev** [innoteq/devpi-slack#3]
Error case does not provide info suitable for debug, because logging is broken.

Environment:
- python==3.6.1
- devpi-server==4.3.0
- devpi-slack==0.1.2

Error:
```python
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/devpi_slack/main.py", line 50, in devpiserver_on_upload_sync
    log.debug(r.content)
  File "/usr/lib/python3.6/site-packages/devpi_server/log.py", line 50, in debug
    self._logout.debug(self._prefix + msg, *args)
TypeError: must be str, not bytes
```

**Comment by max-arnold** [comment]
@apatrushev Thanks for the heads up. Does it work on Python 2.7?

**Comment by max-arnold** [comment]
I did a test on my local machine using the same versions, and see no problems at all:

```
% python --version
Python 3.6.2

% pip freeze | grep devpi
devpi-client==3.0.0
devpi-common==3.1.0
devpi-server==4.3.0
devpi-slack==0.1.2
```

```
Serving on http://localhost:3141
2017-10-03 09:02:06,608 INFO  [req0] POST /testuser/dev/
2017-10-03 09:02:06,611 INFO  [req0] [Wtx5] not re-registering same metadata for testpackage-1.0
2017-10-03 09:02:06,611 INFO  [req0] testuser/dev: got submit release info 'testpackage'
2017-10-03 09:02:06,615 INFO  [req1] POST /testuser/dev/
2017-10-03 09:02:06,625 INFO  [req1] [Wtx5] deleted 'releasefile' link testuser/dev/+f/de5/21a867690af95/testpackage-1.0.tar.gz
2017-10-03 09:02:06,625 INFO  [req1] [Wtx5] set_metadata testpackage-1.0
2017-10-03 09:02:06,626 INFO  [req1] [Wtx5] added 'releasefile' link testuser/dev/+f/6e2/c4a0cee84e5b7/testpackage-1.0.tar.gz
2017-10-03 09:02:06,626 INFO  [req1] [Wtx5] set_metadata testpackage-1.0
2017-10-03 09:02:07,234 INFO  [req1] successfully sent Slack notification: https://hooks.slack.com/services/XXX/YYY/ZZZ
2017-10-03 09:02:07,242 INFO  [req1] [Wtx5] fswriter6: committed: keys: 'testuser/dev/testpackage/.simple','testuser/dev/testpackage/1.0/.config','testuser/dev/+f/de5/21a867690af95/testpackage-1.0.tar.gz','testuser/dev/+f/6e2/c4a0cee84e5b7/testpackage-1.0.tar.gz', files_commit: +files/testuser/dev/+f/6e2/c4a0cee84e5b7/testpackage-1.0.tar.gz, files_del: +files/testuser/dev/+f/de5/21a867690af95/testpackage-1.0.tar.gz
```

**Comment by max-arnold** [comment]
@apatrushev It looks like you have some issues with Slack hook, and the plugin has failed while logging byte-encoded request content (I don't use Python 3 and only tested it using Python 2.7). This should be fixed in `devpi-slack==0.1.3`.

Please try this version and let me know if it works for you.

**Comment by apatrushev** [comment]
Yes. I am using this plugin not for slack in fact and it fails in case of http error. Thank you for urgent fix!
